airbyte-integrations/connectors/source-google-ads/source_google_ads/schemas/ad_group.json

```diff
@@ -90,7 +149,28 @@
       "type": ["null", "string"]
     },
     "ad_group.type": {
-      "type": ["null", "string"]
+      "type": ["null", "string"],
+      "enum": [
```

Comment:
I don't understand the purpose of the ENUM  inside of the data schema. It will cause more problems when the API gives different values:
1) we would  need to update the schema again, once there are updates to the enum values possible
2) it will fail the validation, once there are unidentified values out of the enum list.

Is there a reason behind adding those? If not, let's remove them.

Reply:
I have deleted all enums from schemas.